lib/jsdom/living/nodes/Document-impl.js

```diff
@@ -381,6 +381,11 @@ class DocumentImpl extends NodeImpl {
   }
 
   _createElementWithCorrectElementInterface(name, namespace) {
+    let Constructor = null;
+
+    if (Constructor = this._global.window.customElements.get(name)) {
+      return idlUtils.tryImplForWrapper(new Constructor());
```

Comment:
@TimothyGu 
Hi, I do not ask you support jsdom-wc. Forget about the previous question it was my mistake (this is expected behavior which fallow spec).